oneflow/core/framework/autocast.cpp

```diff
@@ -48,6 +49,26 @@ bool* cache_enabled() {
   return &cache_enabled;
 }
 
+inline Symbol<DType> get_lower_precision_fp_from_device_type(DeviceType device_type) {
+  switch (device_type) {
+    case DeviceType::kCUDA: return get_autocast_gpu_dtype();
+    case DeviceType::kCPU: return get_autocast_cpu_dtype();
+    default:
+      THROW(RuntimeError)
+          << "unknown device type for autocast in get_lower_precision_fp_from_device_type";
+  }
```

Comment:
这里可以改一下，后续还可能支持更多的硬件，把其他硬件暂时都当作gpu。
```suggestion
  if (device_type == DeviceType::kCPU) { return get_autocast_cpu_dtype(); }
  return get_autocast_gpu_dtype();
```